### PR TITLE
Fix scrolling when selecting an issue from About or Done pages (issue #247)

### LINKED ISSUE #247: Scrolling does not occur when activating an issue from the About or Done page
Scrolling an issue into view uses the `#content` selector, which does not exist in `aboutView.js` or `doneView.js`. It looks like it is just trying to scroll to an element that doesn't exist in the view yet. 

I don't know for sure, but on initial inspection I don't think it was introduced by [#208](https://github.com/5calls/5calls/issues/208). 

[About Page](https://i.gyazo.com/e09ff294dbe3322f99020db920438167.mp4)
[Done Page](https://i.gyazo.com/abfe0557dc4a5d7a7c75eccdde77d278.mp4)

### PR DESCRIPTION
**Pull request by bernardwang**
##Summary
[#247](https://github.com/5calls/5calls/issues/247)
Not sure if this is the best way to do it, but it is very fast fix.

### CODE REVIEW

**Review by schraj — APPROVED** [review]
This looks correct to me.

**Review by cdoremus — CHANGES_REQUESTED** [review]
I tried running your 247-pr branch locally and you are missing the 'raf' dependency in package.json which causes the gulp build to fail.
Also, the 247-pr branch is based on your master branch, which you created (and changed) in PR #226. It -- and any other PR -- should be based on an up-to-date master branch refreshed from 5calls/5calls. To do this, add a remote branch to your fork pointing to the 5calls/5calls repo and use that branch to refresh your master branch before you create a branch for your PR. Since this is a small change, I'll let this issue go this time, but please use the proper procedure for any new PR. Thanks.

**Comment by bernardwang** [comment]
@cdoremus I see, next time I will follow that procedure. I incorporated the small changes from , [#249](https://github.com/5calls/5calls/pull/249) but this PR is quite messy, would it be better to close it and redo it properly?

**Comment by nickoneill** [comment]
Nah, we can squash those commits into a single one for clarity. Make sure to use that option when this is ready to merge @cdoremus

**Comment by bernardwang** [comment]
Ok, let me know if theres anything else I should do for this PR. I believe this will need to be merged before [#229](https://github.com/5calls/5calls/pull/229).

File: static/js/components/issues.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const html = require('choo/html');
-const scrollIntoView = require('scroll-into-view');
```
Comment (oderby) [inline static/js/components/issues.js:2]: Can we now remove the dependency on `scroll-into-view` from package.json now? (Assuming this is the last use of it, and we don't re-break everything again, i.e. #255 ?)

**Comment by oderby** [comment]
Latest set of changes looks good to me - @cdoremus any objections to merging?

**Comment by cdoremus** [comment]
@bernardwang @oderby I noticed that when you click on the About link in the footer that the page doesn't scroll to the top. It would be nice to fix that in this PR. Any objections?

**Comment by oderby** [comment]
If @bernardwang thinks it's an easy fix and can get it up soon, then fine by me.

**Comment by bernardwang** [comment]
I didn't find any more instances of `scroll-into-view` when I searched in the project, so it _should_ be safe to remove.

The footer links are just ordinary links, so I added an event listener to attach scrolling. It's easy, but perhaps a bit sloppy. Maybe in the future the footer could be a component rather than being hardcoded in `index.html`? Open to any other suggestions!

**Comment by oderby** [comment]
Yeah, when I agreed with @cdoremus I didn't realize the footer wasn't part of a component. Also, in #251 (wip on i18n support) they are already doing some work to pull the footer into it's own component. Given both of these facts (which make fixing it now rather difficult/ugly, as you point out), I'm actually no longer in favor of increasing the scope of this PR to fix scrolling to top of about page. Sorry for flip-flopping here...

**Comment by cdoremus** [comment]
@bernardwang I agree with @oderby that we should hold off on the About link scrolling for now.

**Comment by bernardwang** [comment]
@oderby What about this option from [#229](https://github.com/5calls/5calls/pull/229#pullrequestreview-28828338)?

**Comment by oderby** [comment]
Seems fine to me. Pulled it down and it seems to work fine @cdoremus thoughts?

Also, @bernardwang this doesn't rebase cleanly onto master branch, due to recent changes - can you merge master and resolve those conflicts and push them up? Don't worry about the mess of the history - as @nickoneill said above, we'll squash when merging this PR in, so it'll get all cleaned up.

**Comment by bernardwang** [comment]
I pulled, but didn't seem to get any conflicts. Is this ok?
I should probably rebase more in the future haha, this history is pretty bad

**Review by cdoremus — APPROVED** [review]
Looks good! Thanks for you patience with our reviews. All commits will be squashed when the merge is done.

**Comment by nickoneill** [comment]
Thanks @bernardwang! This fixes some issues needed for #256 as well, much appreciated!

**Pull request merged**